pcstac/pcstac/filter.py

```diff
@@ -34,6 +33,4 @@ async def _fetch() -> Dict:
                 return queryables
 
         cache_key = f"{CACHE_KEY_QUERYABLES}:{collection_id}"
-        queryables = await cached_result(_fetch, cache_key, request)
-        headers = {"Content-Type": "application/schema+json"}
-        return JSONResponse(queryables, headers=headers)
```

Comment:
should be handled by https://github.com/stac-utils/stac-fastapi/blob/main/stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py#L96-L110